manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1
+Installers:
+- InstallerUrl: https://wsjt.sourceforge.io/downloads/wsjtx-2.6.1-win64.exe

```

Comment:
The x86 installer should also be added - 
> https://wsjt.sourceforge.io/downloads/wsjtx-2.6.1-win32.exe

Reply:
Done